galley/pkg/config/processor/metadata/staticinit.gen.go

```diff
@@ -0,0 +1,35 @@
+// GENERATED FILE -- DO NOT EDIT
+//
+
+package metadata
+
+import (
+	// Pull in all the known proto types to ensure we get their types registered.
+
+	// Register protos in "github.com/gogo/protobuf/types"
+	_ "github.com/gogo/protobuf/types"
```

Comment:
I'm concerned that you're importing all of these in a generic config package.  This means that any galley binary would always depend on k8s, for example.  I don't think that should have to be the case.  Maybe we should make the individual sources responsible for these imports?